countdown.js

```diff
@@ -1,5 +1,15 @@
-function countdown(seconds){
-  // ...
+function countdown(seconds) {
+
+    setInterval(function() {
+        if (seconds >= 0) {
+            console.log(seconds);
+            seconds--;
+        } else {
+            clearInterval();
+        }
+        // no love ternary
+        // seconds >= 0 ? console.log(seconds--) : clearInterval();
```

Comment:
Ha, yeah, not really recommended, especially if you aren't using the returned value.